/*
 * Copyright 2018 The Feast Authors
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     https://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

import "feast/types/Value.proto";

package feast.types;

option java_package = "feast.proto.types";
option java_outer_classname = "EntityKeyProto";
option go_package = "github.com/feast-dev/feast/go/protos/feast/types";

message EntityKey {
    repeated string join_keys = 1;
    repeated feast.types.Value entity_values = 2;
}
